app/src/main/res/xml/preferences_network.xml

```diff
@@ -35,6 +35,12 @@
                 android:key="prefShowDownloadReport"
                 android:summary="@string/pref_showDownloadReport_sum"
                 android:title="@string/pref_showDownloadReport_title"/>
+        <SwitchPreference
+            android:defaultValue="false"
+            android:enabled="true"
+            android:key="prefShowAutoDownloadReport"
+            android:summary="@string/pref_showAutoDownloadReport_sum"
+            android:title="@string/pref_showAutoDownloadReport_title"/>
```

Comment:
Please change the indentation, so it matches other items in the file.

Reply:
Changed.